src/map/script.hpp

```diff
@@ -2155,9 +2155,10 @@ enum e_pcblock_action_flag : uint16 {
 	PCBLOCK_SITSTAND = 0x040,
 	PCBLOCK_COMMANDS = 0x080,
 	PCBLOCK_NPCCLICK = 0x100,
-	PCBLOCK_NPC      = 0x18D,
-	PCBLOCK_EMOTION  = 0x200,
-	PCBLOCK_ALL      = 0x3FF,
+	PCBLOCK_EQUIP    = 0x200,
+	PCBLOCK_NPC      = 0x38D,
+	PCBLOCK_EMOTION  = 0x400,
+	PCBLOCK_ALL      = 0x7FF,
```

Comment:
```suggestion
	PCBLOCK_EMOTION  = 0x200,
	PCBLOCK_EQUIP    = 0x400,
	PCBLOCK_NPC      = 0x58D,
	PCBLOCK_ALL      = 0x7FF,
```
I would prefer to keep the current values, just in case someone hardcoded them.